webapp/src/main/java/com/box/l10n/mojito/rest/asset/AssetWS.java

```diff
@@ -243,6 +244,42 @@ public PollableTask getLocalizedAssetForContentAsync(
     return localizedAssetBodyPollableFuture.getPollableTask();
   }
 
+  /**
+   * Localizes the payload content with translations of a given {@link Asset} in parallel.
+   *
+   * @param assetId
+   * @param multiLocalizedAssetBody
+   * @return
+   */
+  @RequestMapping(value = "/api/assets/{assetId}/localized/parallel", method = RequestMethod.POST)
+  public PollableTask getLocalizedAssetForContentParallel(
+      @PathVariable("assetId") long assetId,
+      @RequestBody MultiLocalizedAssetBody multiLocalizedAssetBody) {
+
+    if (multiLocalizedAssetBody.getAssetId() == null) {
+      multiLocalizedAssetBody.setAssetId(assetId);
+    }
+
+    multiLocalizedAssetBody.setSchedulerName(schedulerName);
+
+    Asset asset = assetRepository.getOne(assetId);
+    meterRegistry
+        .counter(
+            "assetWS.getLocalizedAssetForContentParallel",
+            Tags.of("repositoryId", asset.getRepository().getId().toString()))
```

Comment:
For any new metrics - could we add the repository name in as well?
Having just the ID makes it very hard to parse/interpret the data when using the metrics.